docs/sql/udf.md

```diff
@@ -104,6 +104,7 @@ SQL | Rust
 `TIMESTAMP` | `Timestamp`
 `DATE` | `Date`
 `ARRAY T` | `Vec<T>`
+`VARIANT` | `Variant`
```

Comment:
```suggestion
`VARIANT` | `feldera_sqllib::Variant`
```

same for all types defined in sqllib

Reply:
There's text above that says:
```
The `use feldera_sqllib::*` directive imports the definitions of the
standard Rust types that the compiler uses to implement SQL datatypes.
```

I think the word "standard" should be removed.  They are in fact our custom types.